docs/analysis_plan.qmd

```diff
@@ -12,14 +12,35 @@ date: 2025-07-03
 This document describes the analysis plan to perform a retrospective evaluation of the impact of incorporating wastewater data on the forecast performance of a semi-mechanistic renewal-based model, specifically the model available in the [@wwinference] R package.
 Forecasts of COVID-19 hospital admissions will be generated weekly and evaluated against the final observed hospital admissions data, summed across all age groups in Germany during the 2024-2025 respiratory virus season.
 
+# Introduction
+
+## Need
+Lots of investment in wastewater surveillance, hope is that it can be leveraged to improve situational awareness and potentially forecast performance.
+A number of studies have indicated that wastewater can be a leading indicator of changes in trends [cite Kao, many others].
+Wastewater surveillance for SARS-CoV-2 and Influenza A and B is being routinely collected and made publicly available in Germany, as well as in many other countries throughout the world [cite EU wastewater dashboard, South Africa data, NWSS, New Zealand data, etc.].
+However, it isn't clear how/whether this data is being used for real-time analysis, how existing modeling methods perform in different contexts, what the impact of incorporating wastewater is on model performance, and what other factors in the wastewater surveillance system may impact the model performance.
+
+## Gap
+
+1. Larger gap: Lack of evaluation of impact of wastewater on forecast performance: While a number of models have been developed to use wastewater to forecast hospital admissions, few have been evaluated to assess the impact of incorporating wastewater on forecast performance.
+2. Smaller gap: Unknown how a particular model, developed for one setting, generalises to different settings:
+The `wwinference` R package was developed to produce state-level forecasts of hospital admissions in the United States using state-level hospital admissions data and wastewater data from individual wastewater treatment plants.
+It is not known how this model generalises to a different context with a similar data structure.
+The data in Germany is similar to the U.S. in that they collect hospital admissions data at the state-level and measurements of the concentration of SARS-CoV-2 genomes in individual wastewater treatment plants within the state.
+3. Even smaller gap: Unknown what factors in the wastewater surveillance system impact forecast performance from this particular model.
```

Comment:
also a small gap of how to relate the different catchments?

Reply:
In this case, we are assuming catchments are iid subpopulations (with relative population size acting as weighting) within a state because the data is set up that each catchment area is within a state.